megatron/text_generation_utils.py

```diff
@@ -738,7 +743,18 @@ def generate_samples_interactive(
 
         if torch.distributed.is_initialized() and torch.distributed.get_rank() == 0:
             os.system("clear")
-            raw_text = input("Context prompt >>> ")
+            raw_text = ""
```

Comment:
Can you elaborate on why this code looks like like it treats `\n` as a special case?